src/tests/ftest/deployment/network_failure.py

```diff
@@ -362,8 +379,8 @@ def test_network_failure_isolation(self):
 
         # 2. Create a pool across the four ranks on the two nodes. Use --nsvc=3. We have
         # to provide the size because we're using --ranks.
-        self.add_pool(namespace="/run/pool_size_value/*", create=False)
-        self.pool.target_list.update(target_list)
+        self.add_pool(
+            namespace="/run/pool_size_value/*", create=False, target_list=target_list)
         self.pool.create()
```

Comment:
The `self.pool.create()` isn't necessary, since `create=True` is the default

Reply:
Done. Thanks for pointing out.